.env

```diff
@@ -0,0 +1,8 @@
+MYSQLDB_ROOT_PASSWORD=12345
```

Comment:
.env file should not be pushed to the main branch, consider adding it to .gitignore

Reply:
Agree, but in this project it can be for learning purposes, so not a big deal. The most important thing is to not push any real credentials here, but some dummy ones.